src/utils/wallet/providers/keystone/index.ts

```diff
@@ -0,0 +1,272 @@
+'use client';
+
+import KeystoneSDK from "@keystonehq/keystone-sdk"
+import sdk, { PlayStatus, ReadStatus, SupportedResult } from '@keystonehq/sdk';
+import * as bitcoin from 'bitcoinjs-lib';
+import { Psbt } from "bitcoinjs-lib";
+import { tapleafHash } from 'bitcoinjs-lib/src/payments/bip341';
+import { pubkeyInScript } from 'bitcoinjs-lib/src/psbt/psbtutils';
+import { toXOnly } from 'bitcoinjs-lib/src/psbt/bip371';
+import { PsbtInput } from 'bip174/src/lib/interfaces';
+import { HDKey } from "@scure/bip32";
+
+
+import { WalletProvider, Network, Fees, UTXO } from "../../wallet_provider";
+import { generatePsbtOfBIP322Simple, extractSignatureFromPsbtOfBIP322Simple, NetworkType } from './lib';
+import {
+    getAddressBalance,
+    getTipHeight,
+    getFundingUTXOs,
+    getNetworkFees,
+    pushTx,
+} from "../../../mempool_api";
+
+
+type KeystoneWalletInfo = {
+    extendedPublicKey: string | undefined;
```

Comment:
```suggestion
    extendedPublicKeyHex: string | undefined;
```

nit: just to keep consistent